examples/radial_histogram_equalization.py

```diff
@@ -0,0 +1,140 @@
+"""
+===================================
+Radial Histogram Equalization
+===================================
+
+This example applies the radial histogram equalizing filter (`sunkit_image.radial.rhef`) filter to a sunpy map.
+"""
+
+import astropy.units as u
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import sunpy.data.sample
+import sunpy.map
+
+import sunkit_image.enhance as enhance
+import sunkit_image.radial as radial
+from sunkit_image.utils import equally_spaced_bins
+
+#######################################################################################
+# Let us us the sample AIA image to test the THE filter
+
+# Load AIA image
+aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+# Create radial segments (RHEF should use a dense grid)
+radial_bin_edges = equally_spaced_bins(0, 2, aia_map.data.shape[0] // 2)
+radial_bin_edges *= u.R_sun
+
+# Apply the radial filter to the map
+rhef_map = radial.rhef(aia_map, radial_bin_edges)
+
+
+#######################################################################################
+# It seems that the native sunpy plot routine has a strong effect on the output,
+# so we recommend using imshow.
+
+# Plot the three maps in a single figure with one row and three columns
+fig, axes = plt.subplots(1, 3, figsize=(15, 5), sharex="all", sharey="all", subplot_kw={"projection": aia_map})
+
+# Original AIA map.plot()
+ax = axes[0]
+aia_map.plot(axes=ax, clip_interval=(1, 99.99) * u.percent)
+ax.set_title("Original AIA Map")
+
+# RHEF map.plot()
+ax = axes[1]
+rhef_map.plot(axes=ax, clip_interval=(1, 99.99) * u.percent)
+ax.set_title("THE map.plot()")
+
+# THE imshow(map.data)
+ax = axes[2]
+ax.imshow(rhef_map.data, origin="lower", extent=None, cmap=plt.get_cmap("sdoaia171"))
+ax.set_title("THE imshow()")
```

Comment:
I was worried about this. The codespell changed RHE to THE automatically. It is unfortunately very aggressive. 

In this file: https://github.com/sunpy/sunkit-image/blob/main/_typos.toml we need to add RHE to the list and hope that works!